components/style/media_queries.rs

```diff
@@ -129,8 +129,12 @@ impl ToCss for MediaQuery {
             let (mm, l) = match e {
                 Expression::Width(Range::Min(ref l)) => ("min", l),
                 Expression::Width(Range::Max(ref l)) => ("max", l),
+                Expression::Width(Range::Eq(ref l)) => ("eq", l),
+            };
+            match mm {
```

Comment:
I think this could look cleaner as follows:

```rust
let (mm, l) = match e {
    Expression::Width(Range::Min(ref l)) => ("min-", l),
    Expression::Width(Range::Max(ref l)) => ("max-", l),
    Expression::Width(Range::Eq(ref l)) => ("", l),
};

try!(write!("{}width: ", mm));
```

What do you think?